src/cli.ts

```diff
@@ -124,12 +139,23 @@ export async function runCliServer() {
     walletConfig: {
       id: parsed['wallet-id'],
       key: parsed['wallet-key'],
+      storage: {
```

Comment:
As @dblane-digicatapult mentioned in slack, we should probably put this behind a flag so that we can still run with the default sqlite storage.